Sources/ArcGISToolkit/Common/AttachmentsFeatureElementView.swift

```diff
@@ -177,8 +177,7 @@ struct AttachmentsFeatureElementView: View {
     ///   - newAttachmentName: The new attachment name.
     @MainActor
     func onRename(attachmentModel: AttachmentModel, newAttachmentName: String) async throws -> Void {
```

Comment:
This no longer needs to be `async throws`.
```suggestion
    func onRename(attachmentModel: AttachmentModel, newAttachmentName: String) -> Void {
```